src/main/java/com/coveros/selenified/utilities/Sauce.java

```diff
@@ -48,6 +49,13 @@ public static Boolean isSauce() {
         return hub.contains("ondemand.saucelabs.com");
     }
 
+    public static SauceREST getSauceConnection() throws InvalidHubException {
+        if (!isSauce()) {
+            throw new InvalidSauceException("Sauce hub isn't set");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Sauce hub isn't set" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)